Repositories/authorsRepository.js

```diff
@@ -0,0 +1,35 @@
+const mapper = require('../middleware/booksMapper');
+const logger = require('../middleware/logger');
+const database = require('../database');
+
+function getAuthorById(authorId){
+    logger.logInformation('getAuthorById requested');
+    let query = `SELECT * FROM authors WHERE ID = ${authorId}`;
```

Comment:
Zamień * na listę konkretnych pól. Nigdy ni używaj * bo to stwarza problemy z wyciekiem danych oraz nadmiernym pobieraniem niepotrzebnych danych.